ruff.toml

```diff
@@ -3,4 +3,19 @@ exclude = [".git","__pycache__"]
 
 [lint]
 ignore= ["E741"]
```

Comment:
I'd suggest adding `RUF005` to the ignore list, as this could lead to potential issues with numpy per [this thread](https://github.com/astral-sh/ruff/issues/2142).

This affects the changes where we change from concatenating lists to using the unpacking operator `*`, so if you adopt this suggestion we should also revert those changes.

Reply:
Ah good catch, will do